### DISCUSSION TITLE
Support for Google Coral

### DISCUSSION THREAD

**Discussion by tobias-schuele** [bentoml/BentoML#1788] [General]
Are there any plans to support Google Coral (https://coral.ai/) aka Edge TPU (via TFLite)? Any known roadblocks that would hinder an integration? Thanks!

**Comment by aarnphm** [comment]
Hi @tobias-schuele, thank you for bringing up the topic. This would be under the decision for ARM-based supports for BentoML and we haven't explored this domain yet. We would love to hear more about your usecase with Coral with BentoML.

**Reply by tobias-schuele** [reply]
Thanks for the quick reply, @aarnphm. Concerning use cases: I'm looking for an inference solution on edge devices for local processing of sensor data. Unlike pure cloud-based approaches, scalability is not that much of any issue but rather support for HW accelerators. In that context, ARM support would indeed be important from my perspective although in the case of Coral, there are also use cases building on x86 machines.
